source/ConditionsStore.cpp

```diff
@@ -0,0 +1,456 @@
+/* ConditionsStore.cpp
+Copyright (c) 2020-2022 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+#include "ConditionsStore.h"
+
+#include "DataNode.h"
+#include "DataWriter.h"
+
+using namespace std;
+
+
+
+// Default constructor
+ConditionsStore::DerivedProvider::DerivedProvider(const string &name, bool isPrefixProvider): name(name), isPrefixProvider(isPrefixProvider)
+{
+}
+
+
+
+void ConditionsStore::DerivedProvider::SetGetFun(std::function<int64_t(const std::string&)> newGetFun)
```

Comment:
Judging by other files, the style is to drop the `std::` from here too.
https://github.com/endless-sky/endless-sky/blob/9124602fe322f4ef92ec920a3ec89ca7bd49022a/source/Panel.cpp#L72
```suggestion
void ConditionsStore::DerivedProvider::SetGetFun(function<int64_t(const string&)> newGetFun)
```
There are three more functions below this one that would also need this change.

Reply:
This review-comment is not on the latest commit; this was already fixed.